slam_toolbox:
  ros__parameters:
    mode: "mapping"
    map_frame: map
    odom_frame: odom
    base_frame: base_link
    scan_topic: /scan_leveled
    publish_tf: true
    autostart: true
    resolution: 0.05
    use_scan_matching: true
    minimum_time_interval: 0.1
    transform_publish_period: 0.05

    queue_size: 100
    scan_time: 0.1
    throttle_scans: 1
    transform_timeout: 1.0

    use_scan_matching: false
    minimum_time_interval: 0.1